sql/2021/mobile-web/ect_distribution.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# ECT distribution
+SELECT
+  device,
+  percentile,
+  APPROX_QUANTILES(_4GDensity / (_4GDensity + _3GDensity + _2GDensity + slow2GDensity + offlineDensity), 1000)[OFFSET(percentile * 10)] AS _4GDensity,
+  APPROX_QUANTILES(_3GDensity / (_4GDensity + _3GDensity + _2GDensity + slow2GDensity + offlineDensity), 1000)[OFFSET(percentile * 10)] AS _3GDensity,
+  APPROX_QUANTILES(_2GDensity / (_4GDensity + _3GDensity + _2GDensity + slow2GDensity + offlineDensity), 1000)[OFFSET(percentile * 10)] AS _2GDensity,
+  APPROX_QUANTILES(slow2GDensity / (_4GDensity + _3GDensity + _2GDensity + slow2GDensity + offlineDensity), 1000)[OFFSET(percentile * 10)] AS slow2GDensity,
+  APPROX_QUANTILES(offlineDensity / (_4GDensity + _3GDensity + _2GDensity + slow2GDensity + offlineDensity), 1000)[OFFSET(percentile * 10)] AS offlineDensity
+FROM
+  `chrome-ux-report.materialized.device_summary`,
+  UNNEST(GENERATE_ARRAY(1, 100)) AS percentile
+WHERE
+  date = '2021-07-01'
+GROUP BY
+  device,
+  percentile
+ORDER BY
+  device,
+  percentile
```

Comment:
I ran this query and getting very weird results, where the top percentiles (90-100) for 3G are offsetting the bottom percentiles (1-10) for 4G. Not sure if it's the best way of presenting this data?

Also we don't traditionally report tablet. Though maybe it makes more sense for this chapter?

Reply:
We ran into the same problem last year and ended up scrapping it because of how confusing it was to present (can see the [comment thread here](https://docs.google.com/spreadsheets/d/1DGLY7UEWOlDL5_2dtS_j2eqMjiV-Rw5Fe2y6K6-ULvM/edit#gid=1547689348))

We ideally wanted to find the split of 3g and 4g usage on websites, but aggregating that data across all websites still proves challenging

Reply:
Could you just do them as separate queries looking at only one each time? Might be less confusing then as no offsetting.